packages/components/NEW-COMPONENT-CHECKLIST.md

```diff
@@ -76,9 +76,13 @@ The engineering checklist has six parts: creating the feature branch, component
 ### Component Creation
 
 - [ ] create new branch from main for the component (`git checkout -b USER/COMPONENT-NAME`)
+  - Note: There is no standardized branch naming convention.
+  - Suggestion: Use Jira ticket number together with a brief description. (Including the ticket number may allow us to automatically link branches to tickets and PRs in the future.)
+  - Example Branch Name: "hds-123-fix-typo".
 - [ ] create new component using a blueprint
   - `ember generate hds-component COMPONENT-NAME`
   - if it's a child component, then `hds/COMPONENT_NAME/CHILD_NAME`
+  - (Note: Many of the below mentioned files will be automatically generated.)
```

Comment:
Again, let me know if this ok or if I should change or move to new PR.

Reply:
@MelSumner Would you like me to undo this change and move to a new PR?

Reply:
@didoo Do you think it's helpful to include this note? I can delete it if it's not. I deleted the other text I added on branch naming since Melanie didn't seem to agree with it.

Reply:
OK to leave it as is.